cmake/defaults/Packages.cmake

```diff
@@ -99,6 +99,8 @@ else()
     message(STATUS "Skipping OpenVDB support")
 endif()
 
+find_package(MaterialX REQUIRED)
```

Comment:
@bsavery, should we include MaterialX as submodule or expect the user to provide it, what do you think?

Reply:
I'm going to make it a submodule because on Linux we need to make sure that `_glibcxx_use_cxx11_abi` is set to 0 when building Houdini plugin.